targets/liberty-apis/pom.xml

```diff
@@ -19,228 +19,234 @@
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.basics</artifactId>
-      <version>1.2.12</version>
+      <version>1.2.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.clusterMember</artifactId>
-      <version>1.0.12</version>
+      <version>1.0.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.collectiveController</artifactId>
-      <version>1.5.12</version>
+      <version>1.5.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.config</artifactId>
-      <version>1.2.12</version>
+      <version>1.2.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.connectionpool</artifactId>
-      <version>1.0.12</version>
+      <version>1.0.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.constrainedDelegation</artifactId>
-      <version>1.0.12</version>
+      <version>1.0.13</version>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.distributedMap</artifactId>
-      <version>2.0.12</version>
+      <version>2.0.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.dynamicRouting</artifactId>
-      <version>1.0.12</version>
+      <version>1.1.13</version>

```

Comment:
Yes that's correct:
http://search.maven.org/#search|gav|1|g%3A%22com.ibm.websphere.appserver.api%22%20AND%20a%3A%22com.ibm.websphere.appserver.api.dynamicRouting%22